sanic/request.py

```diff
@@ -304,19 +305,21 @@ def parse_multipart_form(body, boundary):
             if form_header_field == 'content-disposition':
                 if 'filename' in form_parameters:
                     file_name = form_parameters['filename']
-                field_name = form_parameters.get('name')
+                field_name = form_parameters['name']
             elif form_header_field == 'content-type':
-                file_type = form_header_value
+                content_type = form_header_value
+                if 'charset' in form_parameters:
+                    content_charset = form_parameters['charset']
```

Comment:
let's just do `content_charset = form_parameters.get('charset')` and remove the `if 'charset' in form_parameters:`

Reply:
then it will default to `utf-8` if not specified

Reply:
I'll change it :)